src/renderer/bootstrap.tsx

```diff
@@ -90,9 +91,41 @@ export async function bootstrap(App: AppComponent) {
   });
   render(<>
     {isMac && <div id="draggable-top" />}
-    <App />
+    {DefaultProps(App)}
   </>, rootElem);
 }
 
+const defaultTheme = createMuiTheme({
```

Comment:
What is this and why styling is done via "MuiTheme"?

Reply:
This is how we change the defaults for material ui. This basically adds a new `<style>` to our DOM at the bottom of the `<head>` so it has the highest priority.

Reply:
I think we have done this differently in the past (through scss). @aleksfront @ixrock  wdyt about this?

Reply:
We didn't get any feedback on this... at least move this styling to a separate file and import it here (just to keep things cleaner).

Reply:
okay